src/identity-providers/add/DescriptorSettings.tsx

```diff
@@ -0,0 +1,428 @@
+              }}
+              selections={value}
+              variant={SelectVariant.single}
+            >
+              <SelectOption
+                key={0}
+                data-testid="subjectNameId-option"
+                value={t("subjectNameId")}
+                isPlaceholder
+              />
+              <SelectOption
+                key={1}
+                data-testid="attributeName-option"
+                value={t("attributeName")}
+              />
+              <SelectOption
+                key={2}
+                data-testid="attributeFriendlyName-option"
+                value={t("attributeFriendlyName")}
+              />
+            </Select>
+          )}
+        ></Controller>
+      </FormGroup>
+
+      <SwitchField
+        field="config.postBindingResponse"
+        label="httpPostBindingResponse"
+        isReadOnly={readOnly}
+      />
+
+      <SwitchField
+        field="config.postBindingAuthnRequest"
+        label="httpPostBindingAuthnRequest"
+        isReadOnly={readOnly}
+      />
+
+      <SwitchField
+        field="config.postBindingLogout"
+        label="httpPostBindingLogout"
+        isReadOnly={readOnly}
+      />
+
+      <SwitchField
+        field="config.wantAuthnRequestsSigned"
+        label="wantAuthnRequestsSigned"
+        isReadOnly={readOnly}
+      />
+
+      {wantAuthnSigned === "true" && (
+        <>
+          <FormGroup
+            label={t("signatureAlgorithm")}
+            labelIcon={
+              <HelpItem
+                helpText={th("signatureAlgorithm")}
+                forLabel={t("signatureAlgorithm")}
+                forID="kc-signatureAlgorithm"
+              />
+            }
+            fieldId="kc-signatureAlgorithm"
+          >
+            <Controller
+              name="config.signatureAlgorithm"
+              defaultValue="RSA_SHA256"
+              control={control}
+              render={({ onChange, value }) => (
+                <Select
+                  toggleId="kc-signatureAlgorithm"
+                  onToggle={() =>
+                    setSignatureAlgorithmDropdownOpen(
+                      !signatureAlgorithmDropdownOpen
+                    )
+                  }
+                  isOpen={signatureAlgorithmDropdownOpen}
+                  onSelect={(_, value) => {
+                    onChange(value as string);
+                    setSignatureAlgorithmDropdownOpen(false);
+                  }}
+                  selections={value}
+                  variant={SelectVariant.single}
+                >
+                  <SelectOption key={0} value="RSA_SHA1" />
+                  <SelectOption key={1} value="RSA_SHA256" isPlaceholder />
+                  <SelectOption key={2} value="RSA_SHA256_MGF1" />
+                  <SelectOption key={3} value="RSA_SHA512" />
+                  <SelectOption key={4} value="RSA_SHA512_MGF1" />
+                  <SelectOption key={5} value="DSA_SHA1" />
```

Comment:
I don't think it's needed to give these `SelectOption` a `key` prop, as they are statically populated.
```suggestion
                  <SelectOption value="RSA_SHA1" />
                  <SelectOption value="RSA_SHA256" isPlaceholder />
                  <SelectOption value="RSA_SHA256_MGF1" />
                  <SelectOption value="RSA_SHA512" />
                  <SelectOption value="RSA_SHA512_MGF1" />
                  <SelectOption value="DSA_SHA1" />
```

Reply:
yep, this was fixed last iteration, ty